src/Nethermind/Nethermind.Evm/Tracing/GethStyle/Custom/JavaScript/GethLikeJavaScriptTxTracer.cs

```diff
@@ -99,12 +99,12 @@ public override void ReportAction(long gas, UInt256 value, Address from, Address
             : new Log.Contract(from, to, value, isAnyCreate ? null : input);
     }
 
-    public override void StartOperation(int depth, long gas, Instruction opcode, int pc, bool isPostMerge = false)
+    public override void StartOperation(in ExecutionEnvironment env, long gas, Instruction opcode, int pc)
     {
         _log.pc = pc;
         _log.op = new Log.Opcode(opcode);
         _log.gas = gas;
-        _log.depth = depth;
+        _log.depth = env.CallDepth + 1;
```

Comment:
maybe on `ExecutionEnvironment` we should have extension method called `GetGethTraceDepth()` or something?